L1Trigger/L1THGCal/src/backend/HGCalMulticlusteringHistoImpl.cc

```diff
@@ -582,7 +540,7 @@ std::vector<l1t::HGCalMulticluster> HGCalMulticlusteringHistoImpl::clusterSeedMu
 
             double seedWeight = 1;
             if ( cluster_association_strategy_ == EnergySplit) seedWeight = energy.second/totalTargetSeedEnergy;
-            if( mapSeedMulticluster[energy.first ].size()==0) {
+            if( mapSeedMulticluster[energy.first ].empty()) {
```

Comment:
Sorry, I was confused. This is not the size of the map, this is the size of the multicluster in the map. So size()==0 was fine since there is no cluster::empty() method.
Which is why the code checker didn't correct it....

Reply:
Yes sorry I should have compiled and checked myself.